pkg/security/probe/process_killer.go

```diff
@@ -459,13 +613,27 @@ const (
 	executableDisarmerType disarmerType = "executable"
 )
 
+type processContext struct {
```

Comment:
I would rather call it killContext

Reply:
yeah I already though about renaming it :) will do, thanks

Reply:
done